src/config/parser.rs

```diff
@@ -460,6 +461,51 @@ mod tests {
         )
     }
 
+    #[test]
+    fn variant_trailing_comma() {
+        success(
+            &toklist![
+                TokType::LBracket,
+                "a",
+                TokType::Comma,
+                TokType::RBracket,
+                TokType::Semicolon
+            ],
+            &[Entry {
+                left: Spec {
+                    string: None,
+                    spectype: SpecType::variant_expr(vec![Spec::from("a")], None),
+                },
+                right: None,
+            }],
+        )
+    }
+
+    #[test]
+    fn match_trailing_comma() {
+        success(
+            &toklist![
+                TokType::LBrace,
+                "linux",
+                TokType::Colon,
+                "a",
+                TokType::Comma,
+                TokType::RBrace,
+                TokType::Semicolon
+            ],
+            &[Entry {
+                left: Spec {
+                    string: None,
+                    spectype: SpecType::match_expr(
+                        vec![(ExprType::Linux.into(), Spec::from("a"))],
+                        None,
+                    ),
+                },
+                right: None,
+            }],
+        )
+    }
```

Comment:
Add a comment to each of these test functions explaining that trailing commas are valid syntax and should be successfully parsed.